numpy/typing/tests/data/reveal/ma.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import Any
+
+import numpy as np
+
+from typing_extensions import assert_type
+
+arr: np.ma.MaskedArray[Any, np.dtype[np.float64]] = np.ma.masked_array([[1.0, 2.0]], mask=[True, False])
+result = np.ma.min(arr, axis=None)
+assert_type(result, np.float64)
```

Comment:
by inlining the `result`, you avoid changing the type `result` later on, which technically isn't valid (mypy allowing this is a bug IMHO).

```suggestion
assert_type(np.ma.min(arr, axis=None), np.float64)
```